README.md

```diff
@@ -1 +1,141 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+- Getting Started
+- Authors
+- License
+<!-- TABLE OF CONTENTS -->
```

Comment:
### Kindly update your README with the following:
   - [ ] All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. 
   - [ ] The only optional section is FAQ.
   - [ ] You should delete the live demo section if you decide not to deploy the application.
   - [ ] Make sure to add the section Future Features, including 1-3 potential features to be added.
   - [ ] You should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project.
   - [ ] It is your choice regarding how much into the details they go in each section of the README.
   - [ ] You should remove the **template logo**. Optionally, you can replace it with your own logo if you have one.
   - [ ] Kindly update the link to the MIT license at the bottom of README as now when I press it I get 404.
   - [ ] You don't need to include the objectives, general requirements, and requirements or a definition of the languages in the Built with section.

Reply:
Thanks @gemmen29, all suggestions are fully implemented.